tools/misc/cubemanip

```diff
@@ -1,280 +1,365 @@
 #!/usr/bin/env python3
-#------------------------------------------------------------------------------#
-#  DFTB+: general package for performing fast atomistic simulations            #
-#  Copyright (C) 2006 - 2023  DFTB+ developers group                           #
-#                                                                              #
-#  See the LICENSE file for terms of usage and distribution.                   #
-#------------------------------------------------------------------------------#
-
-from numpy import oldnumeric as num
-from numpy import linalg as linal
-import math
-import sys
-import re
+#--------------------------------------------------------------------------------------------------#
+#  DFTB+: general package for performing fast atomistic simulations                                #
+#  Copyright (C) 2006 - 2023  DFTB+ developers group                                               #
+#                                                                                                  #
+#  See the LICENSE file for terms of usage and distribution.                                       #
+#--------------------------------------------------------------------------------------------------#
+
+"""Script to evaluate expressions on cube files"""
+
 import argparse
+from dataclasses import dataclass, field
+import re
+import sys
+import numpy as np
+import numpy.linalg as la
 
 SCRIPT_NAME = "cubemanip"
 
-# Tolerance for positions to be treated as equal
-tolerance = 1.0e-5
+# tolerance for positions to be treated as equal
+TOLERANCE = 1.0e-5
```

Comment:
Comment probably outside scope of this PR: Any reason for the specific value, and why not a user supplied argument with a default?

Reply:
Sure, we could do of course. The usual case is, however, that you combine cube-files from the same waveplot calculations, where the tolerance above should be OK-ish.